wilsonloop.hh

```diff
@@ -66,6 +98,26 @@ template<class Group> void compute_all_loops(gaugeconfig<Group> &U, std::string
   return;
 }
 
+template<class Group> void compute_all_paths(gaugeconfig<Group> &U, std::string const &path, 
```

Comment:
what does this function do?

Reply:
It should compute the Wilson-loops W(R, T) for several T and average over different ways to go a distance R in the lattice

Reply:
reflect that in the function name?